src/js/components/Sidebar.js

```diff
@@ -228,6 +168,39 @@ var Sidebar = React.createClass({
     return <ul className="sidebar-menu">{groupMenuItems}</ul>;
   },
 
+  getGroupSubmenu(children) {
+    let {pathname} = this.props.location;
+    let isChildActive = false;
+
+    let menuItems = children.reduce(function (children, currentChild) {
+      let isActive = pathname.startsWith(currentChild.path);
```

Comment:
Instead of `startsWith` should we have more robust check like splitting on `/` and matching the array indices from 0 -> number of url "parts"? That would prevent false positives.